### ISSUE TITLE
Error: add site dependencies: load resources: loading templates in summary.html:18: unclosed action

### ISSUE THREAD

**Issue by teotenn** [opera7133/Blonde#128]
Hi, I've been using the theme Blonde for longer than a year now, and recently I started getting the error related to `postcss-cli`. Thus, I have just updated the theme and follow again all the instructions that you have now in the README. 

Now I have a different error:

```
 Blonde@0.0.2 start
> concurrently npm:watch:*

[watch:*tw] 
[watch:*tw] > Blonde@0.0.2 watch:tw
[watch:*tw] > tailwindcss -i ./themes/Blonde/assets/css/main.css -o ./themes/Blonde/assets/css/style.css --watch
[watch:*tw] 
[watch:*hugo] 
[watch:*hugo] > Blonde@0.0.2 watch:hugo
[watch:*hugo] > hugo server
[watch:*hugo] 
[watch:*hugo] Error: add site dependencies: load resources: loading templates: "/home/teoten/Code/WebApps/Rwhitedwarf-blog/themes/Blonde/layouts/_default/summary.html:18:1": parse failed: template: _default/summary.html:18: unclosed action
[watch:*hugo] npm run watch:hugo exited with code 255
[watch:*tw] 
[watch:*tw] Rebuilding...
[watch:*tw] Done in 3694ms.
```
Appears when running `npm run start` locally on hugo v 0.80.0. When run on my server on render.com with Hugo v0.104.1 the error is more complex and longer: if I run `npm run start` the process would run endlessly with no end and no rendering. If instead I executed `hugo --gc --minify" it ends with `Build failed` and the last part of the error is

```
Error: Error building site: failed to render pages: render of "taxonomy" failed: "/opt/render/project/src/themes/Blonde/layouts/_default/baseof.html:7:4": execute of template failed: template: _default/list.html:7:4: executing "_default/list.html" at <partial "head.html" .>: error calling partial: "/opt/render/project/src/layouts/partials/head.html:17:51": execute of template failed: template: partials/head.html:17:51: executing "partials/head.html" at <postCSS (dict "config" "./assets/css/postcss.config.js")>: error calling postCSS: no Resource provided in transformation
```

Any ideas what can I be doing wrong?

**Comment by teotenn** [comment]
I have updated Hugo locally and now it gives me similar error than render when I run `npm run build`

```
one in 3198ms.
Start building sites … 
hugo v0.104.2+extended linux/amd64 BuildDate=unknown
ERROR 2022/10/04 22:18:13 render of "page" failed: "/home/teoten/Code/WebApps/Rwhitedwarf-blog/themes/Blonde/layouts/_default/baseof.html:7:4": execute of template failed: template: _default/single.html:7:4: executing "_default/single.html" at <partial "head.html" .>: error calling partial: "/home/teoten/Code/WebApps/Rwhitedwarf-blog/layouts/partials/head.html:17:51": execute of template failed: template: partials/head.html:17:51: executing "partials/head.html" at <postCSS (dict "config" "./assets/css/postcss.config.js")>: error calling postCSS: no Resource provided in transformation
ERROR 2022/10/04 22:18:13 render of "page" failed: "/home/teoten/Code/WebApps/Rwhitedwarf-blog/themes/Blonde/layouts/_default/baseof.html:7:4": execute of template failed: template: _default/single.html:7:4: executing "_default/single.html" at <partial "head.html" .>: error calling partial: "/home/teoten/Code/WebApps/Rwhitedwarf-blog/layouts/partials/head.html:17:51": execute of template failed: template: partials/head.html:17:51: executing "partials/head.html" at <postCSS (dict "config" "./assets/css/postcss.config.js")>: error calling postCSS: no Resource provided in transformation
ERROR 2022/10/04 22:18:13 render of "page" failed: "/home/teoten/Code/WebApps/Rwhitedwarf-blog/themes/Blonde/layouts/_default/baseof.html:7:4": execute of template failed: template: post/single.html:7:4: executing "post/single.html" at <partial "head.html" .>: error calling partial: "/home/teoten/Code/WebApps/Rwhitedwarf-blog/layouts/partials/head.html:17:51": execute of template failed: template: partials/head.html:17:51: executing "partials/head.html" at <postCSS (dict "config" "./assets/css/postcss.config.js")>: error calling postCSS: no Resource provided in transformation
ERROR 2022/10/04 22:18:13 render of "page" failed: "/home/teoten/Code/WebApps/Rwhitedwarf-blog/themes/Blonde/layouts/_default/baseof.html:7:4": execute of template failed: template: post/single.html:7:4: executing "post/single.html" at <partial "head.html" .>: error calling partial: "/home/teoten/Code/WebApps/Rwhitedwarf-blog/layouts/partials/head.html:17:51": execute of template failed: template: partials/head.html:17:51: executing "partials/head.html" at <postCSS (dict "config" "./assets/css/postcss.config.js")>: error calling postCSS: no Resource provided in transformation
Error: Error building site: failed to render pages: render of "page" failed: "/home/teoten/Code/WebApps/Rwhitedwarf-blog/themes/Blonde/layouts/_default/baseof.html:7:4": execute of template failed: template: _default/single.html:7:4: executing "_default/single.html" at <partial "head.html" .>: error calling partial: "/home/teoten/Code/WebApps/Rwhitedwarf-blog/layouts/partials/head.html:17:51": execute of template failed: template: partials/head.html:17:51: executing "partials/head.html" at <postCSS (dict "config" "./assets/css/postcss.config.js")>: error calling postCSS: no Resource provided in transformation
Total in 216 ms
```
That is with your default `config.toml` file, my current config file can be found here https://github.com/RWhiteDwarf/blog with all other details of my build

**Comment by opera7133** [comment]
Your `layouts/partials/head.html` should be this code.
```
{{ $styles := resources.Get "css/styles.css" | postCSS (dict "config" "./assets/css/postcss.config.js") }}
```
This code is a bit out of date.
Please replace it with the following code.
```
{{ $styles := resources.Get "css/style.css" | postCSS }}
```
I have tested and confirmed that it works.

**Comment by teotenn** [comment]
It did not help. What is weird is that I cloned the latest commit of the overall website that was stable, when I tried to build from there it sent me a different error. I applied there the change you are suggesting here and also could not build. 

Anyway thanks a lot for your support. I changed the theme for something more simple and less dependant on npm which is anyway what I need. I'm closing this issue.
